### PR TITLE
Konveyor Addon

### PR DESCRIPTION
**Pull request by ariveroi**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/addons/konveyor.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+# Konveyor Add-On for Amazon EKS Blueprints
+
+Streamline the modernization of your application portfolio to Kubernetes. The Konveyor tool gives you insight throughout the migration process - whether you're making decisions at the portfolio or application level.
```
Comment (elamaran11) [inline docs/addons/konveyor.md:3]: Same feedback as patterns repo. Start with What is Konveyor and what is its value proposition to customers? Check our other partner addons for example

File: docs/addons/konveyor.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+- Konveyor Hub: Central interface from where you manage your application portfolio and integrate with other Konveyor tools.
+
+- Categorize and group applications by different dimensions (pre-packaged or custom) aligned with technical criteria or your organization structure.
+
+- Assess applications containerization suitablity and risks assessment.
+
+- Assign priority, assess estimated migration effort, and define optimal migration strategy for individual applications.
+
+- Evaluate required changes for Java applications containerization through automated analysis (pre-packaged or custom rules).
+
```
Comment (elamaran11) [inline docs/addons/konveyor.md:16]: Architecture diagram on integration with EKS helps customers to relate it better.

File: docs/addons/konveyor.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+- [npm](https://docs.npmjs.com/cli/v8/commands/npm-install)
+- [tsc](https://www.typescriptlang.org/download)
+- [make](https://www.gnu.org/software/make/)
+
+Let’s start by setting the account and region environment variables:
+
+```sh
+ACCOUNT_ID=$(aws sts get-caller-identity --query 'Account' --output text)
+AWS_REGION=$(aws configure get region)
+```
```
Comment (elamaran11) [inline docs/addons/konveyor.md:38]: Where is the secrets manager command if there is a secret.

File: docs/addons/konveyor.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+
+export interface KonveyorConstructProps extends StackProps {
+  account: string;
+  region: string;
+  parentDomain: string;
+  konveyorLabel: string;
+  hostedZoneId: string;
+  certificateResourceName: string;
+}
+
```
Comment (elamaran11) [inline docs/addons/konveyor.md:118]: jsdoc for classes helps

File: docs/addons/konveyor.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+```
+
+## Log in
+
+Once the deployment ends navigate to
+
+`https://<subdomain>.<parent-domain>`
+
+And enter the default admin credentials:
+
```
Comment (elamaran11) [inline docs/addons/konveyor.md:203]: have screenshots of console.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@ariveroi Similar comments to Patterns repo but also have more :
1. Since this looks like partner product, Please have partner commenting on the repo for ongoing maintenance.
2. Please share license keys for validation.
3. Documentation needs to be updated per comments for completeness.
4. The addon is not added to addon list documentation [here](https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/docs/addons/index.md).
5. Please review our partner [extensions](https://aws-quickstart.github.io/cdk-eks-blueprints/extensibility/#partner-extensions) guide once.

**Comment by cbellone** [comment]
hello @elamaran11

This is Celestino Bellone, I am an employee of Claranet Switzerland, and I am one of the maintainers of https://github.com/claranet-ch/konveyor-eks-blueprint-addon .

As requested, this is to confirm that Claranet will support the ongoing maintenance of the Konveyor EKS Addon.

Let me know if you need anything else from our side.

Thanks,
Celestino

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@ariveroi Nice work. Update to `index.md` doc file is missing. Rest looks good atm.

**Review by freschri — DISMISSED** [review]

**Review by elamaran11 — COMMENTED** [review]
@ariveroi Have a question

**Review by elamaran11 — APPROVED** [review]
LGTM.

**Pull request merged**